src/panels/config/ha-panel-config.ts

```diff
@@ -548,6 +548,13 @@ class HaPanelConfig extends SubscribeMixin(HassRouterPage) {
             "./integrations/integration-panels/thread/thread-config-panel"
           ),
       },
+      bluetooth: {
+        tag: "bluetooth-device-page",
+        load: () =>
+          import(
+            "./integrations/integration-panels/bluetooth/bluetooth-device-page"
```

Comment:
```suggestion
            "./integrations/integration-panels/bluetooth/bluetooth-config-panel"
```
The other integration pages are named `config-panel`